code/datums/mood_events/generic_positive_events.dm

```diff
@@ -29,11 +29,6 @@
 	mood_change = 1
 	timeout = 2 MINUTES
 
-/datum/mood_event/tailpulled
-	description = "I love getting my tail pulled!"
-	mood_change = 1
-	timeout = 2 MINUTES
```

Comment:
There should be a negative moodlet at the very minimum if you want it to not be weird